hail/python/hailtop/auth/auth.py

```diff
@@ -101,10 +101,11 @@ async def async_copy_paste_login(copy_paste_token: str, namespace: Optional[str]
             raise_for_status=True,
             timeout=aiohttp.ClientTimeout(total=5),
             headers=headers) as session:
-        async with await request_retry_transient_errors(
```

Comment:
This should be an `httpx.ClientSession`

Reply:
Great catch. I removed another case of aiohttp.ClientSession from job.py as well. It now only appears in httpx.py.